From 053b63b5bbd8437926fb88ce7797d6f456772137 Mon Sep 17 00:00:00 2001
From: Erik Massop <e.massop@hccnet.nl>
Date: Thu, 19 Jun 2014 01:11:07 +0200
Subject: [PATCH 2/5] tr portability

Squashed commit of the following:

commit c5db9a7871250f97f69f63b6803e866732c09ae5
Author: thome <thome@e5c1114b-a573-4582-9dac-f72f410959ce>
Date:   Tue Oct 22 17:02:39 2013 +0000

    avoid using tr -d -c 0-9, which is not portable enough...

    git-svn-id: svn://scm.gforge.inria.fr/svnroot/gf2x/trunk@161 e5c1114b-a573-4582-9dac-f72f410959ce
---
 config/acinclude.m4 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/config/acinclude.m4 b/config/acinclude.m4
index 05cc9b9..9830d93 100644
--- a/config/acinclude.m4
+++ b/config/acinclude.m4
@@ -46,7 +46,9 @@ AC_DEFUN([RUNTIME_ULONG_BITS],[
     AC_CACHE_CHECK([the number of bits in an unsigned long],
         [gf2x_cv_ulongbits],[
         AC_RUN_IFELSE([WORDSIZE_CODE()],[
-            detected=`cat conftest.out | tr -d -c 0-9`
+            # see bug #15631 and autoconf manual about tr.
+            # detected=`cat conftest.out | tr -d -c 0-9`
+            detected=`cat conftest.out`
             if test x$detected = x ; then
                 AC_MSG_ERROR([test program failed])
             else
-- 
2.0.0

